pkg/api/types.go

```diff
@@ -1763,7 +1767,9 @@ type ServiceSpec struct {
 	// This field will be ignored if the cloud-provider does not support the feature.
 	LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
 
-	// Optional: Supports "ClientIP" and "None".  Used to maintain session affinity.
+	// Optional, used to maintain session affinity.
+	// Supports "ClientIP", "RequireNodeLocal" and "None". The second makes
+	// most sense with `Type: ClusterIP`. Defaults to None.
```

Comment:
Note that RequireNodeLocal is alpha.  In fact, since this is a constant. we should probably call it AlphaRequireNodeLocal for now and then require a change later for end users (you can leave the internal constant as RequireNodeLocal, but the value for the public API should be Alpha*).